src/dkg_hybriddkg.erl

```diff
@@ -16,18 +16,18 @@
           t :: pos_integer(),
           u :: erlang_pbc:element(),
           u2 :: erlang_pbc:element(),
-          vss_map = #{} :: vss_map(),
-          vss_results = #{} :: vss_results(),
-          qbar = [] :: qset(),
-          qhat = [] :: qset(),
-          rhat = [] :: rhat(),
-          mbar = [] :: mbar(),
-          elq = #{} :: elq(),
-          rlq = #{} :: rlq(),
-          lc_flag = false :: boolean(),
+          shares_map = #{} :: shares_map(),
+          shares_results = #{} :: shares_results(),
+          shares_seen = [] :: node_set(),
+          shares_acked = [] :: node_set(),
+          echo_proofs = [] :: echo_proofs(),
+          echo_counts = #{} :: echo_counts(),
+          ready_proofs = [] :: ready_proofs(),
+          ready_counts = #{} :: ready_counts(),
           leader :: pos_integer(),
-          l_next :: pos_integer(),
-          lc_map = #{} :: lc_map()
+          leader_cap :: pos_integer(),
```

Comment:
Just curious, why is this leader_cap instead of leader_next or something, is it because of the `^` (hat) symbol or does it convey some other meaning?

Reply:
the reason that it's there is inherited from PBFT, it's intended as a cap for the size of the next leader proposed so that you can't do stuff like sending an `Lbar` of MAXINT or whatever and attack the protocol that way.

Reply:
Ah I see, makes sense to keep it as a cap then